plugins/woocommerce-admin/client/task-lists/fills/experimental-shipping-recommendation/shipping-recommendation.tsx

```diff
@@ -76,6 +78,10 @@ export const ShippingRecommendation: React.FC<
 		setPluginsToActivate( remainingPlugins );
 	}, [ activePlugins, isJetpackConnected, isResolving, pluginsToActivate ] );
 
+	const prefetchedJetpackAuthData = useSelect( ( select ) => {
```

Comment:
Prefetching the Jetpack auth URL here and then passing it to the `<Connect>` component so users don't have to wait for the "Connect" to finish retrieving the auth URL from the REST API.